pkg/repository/prisma/tenant.go

```diff
@@ -395,3 +441,13 @@ func (r *tenantEngineRepository) RebalanceAllTenantWorkerPartitions(ctx context.
 func (r *tenantEngineRepository) RebalanceInactiveTenantWorkerPartitions(ctx context.Context) error {
 	return r.queries.RebalanceInactiveTenantWorkerPartitions(ctx, r.pool)
 }
+
+func getPartitionName() pgtype.Text {
+	hostname, ok := os.LookupEnv("HOSTNAME")
```

Comment:
do we want to put this on the server config file?

Reply:
not at the moment, as this will only be used internally for multi-container setups where hostname is almost always set.